package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "html",
+    "name": "it-conference",
```

Comment:
Seems unrelated, why is this needed?

Reply:
No clue why that changed. Normally when I checkout branches I do
 ```
Composer install && npm install

```
I'll change it back to html